pull_from_solr.rb

```diff
@@ -54,7 +54,9 @@
   bar.advance
 
   result["response"]["docs"].each do |doc|
-    outfile.puts(doc.to_json)
+    json_string = doc.to_json
+    modified_json_string = json_string.gsub(/,"_version_":[0-9]+/, '')
+    outfile.puts(modified_json_string)
```

Comment:
A small adjustment to make this less brittle and possibly a little more readable, by deleting the version key from the hash rather than replacing it out of a string:

```suggestion
    doc.delete('_version_')
    outfile.puts(doc.to_json)
```

Reply:
Yeah, that's fine. This is what I had originally but thought the performance was better with .gsub. Maintainability is good too.